src/goto-programs/resolve_inherited_component.cpp

```diff
@@ -68,7 +68,7 @@ resolve_inherited_componentt::inherited_componentt
     }
 
     const class_hierarchyt::idst &parents=
-      class_hierarchy.class_map[current_class].parents;
+      class_hierarchy.class_map.at(current_class).parents;
```

Comment:
While I never had an exception thrown, in the spirit of keeping the code functionally the same it might be better to do a find on the map, then only do anything more if that succeeds.

Think that would be prudent, or leave it as-is and add the precondition?